docs/guides/release.md

```diff
@@ -0,0 +1,44 @@
+# release binaries
+
+## Table of Contents
+* [Introduction](#intorduction)
+* [Generate](#generate)
+* [Add architecture](#add architecture)
+* [Remove architecture](#remove architecture)
+
+## Introduction
+In this file I will describe, how we can build the release binaries. 
+In the current state we will build the release for the following architectures:
+- armV6
+- armV7
+- x86\_64
+
+We are using a seperate release script on the build/tool directory.
+
+## Generate
+To generate a current release you can build all the files from hand or using this script. You can execute from the root direcotry of this repository via this command:
+```bash
+bash build/tools/release.sh $version $destinationDir
+```
+
+In the destination directory you fill find the following files:
+- keadm-$version-linux-x86\_64
+- keadm-$version-linux-armV6
+- keadm-$version-linux-armV7
+- edge-$version-linux-x86\_64
+- edge-$version-linux-armV6
+- edge-$version-linux-armV7
+- kubeedge-$version-linux-x86\_64
+- kubeedge-$version-linux-armV6
+- kubeedge-$version-linux-armV7
+- edgesite-$version-linux-x86\_64
+- edgesite-$version-linux-armV6
+- edgesite-$version-linux-armV7
+- the checksum files to every builded binary package
```

Comment:
Can we create a single checksum file which is having the checksums for all the binaries separated by newlines ?
Maybe too many checksum files is not very user friendly ?

Reply:
Yes of course, I will use checksum_$arch.txt as filename